core/modules/taxonomy/taxonomy.install

```diff
@@ -403,6 +403,23 @@ function taxonomy_update_1009() {
   }
 }
 
+/**
+ * Clean up extraneous language-related configuration setting.
+ */
+function taxonomy_update_1010() {
+  $config_names = config_get_names_with_prefix('field.bundle.taxonomy_term.');
+  foreach ($config_names as $config_name) {
+    $config = config($config_name);
+    $extra_fields_display_langcode = $config->get('extra_fields.display.language');
+    $extra_fields_form_langcode = $config->get('extra_fields.form.language');
+    $config->set('extra_fields.display.langcode', $extra_fields_display_langcode);
+    $config->set('extra_fields.form.langcode', $extra_fields_form_langcode);
+    $config->clear('extra_fields.display.language');
+    $config->clear('extra_fields.form.language');
```

Comment:
Let's add a bit of defensive checking here in the event that config has been imported that is already using `langcode` instead of `language`:
```suggestion
    $extra_fields_display_langcode = $config->get('extra_fields.display.language');
    if (isset($extra_fields_display_langcode)) {
      $config->set('extra_fields.display.langcode', $extra_fields_display_langcode);
      $config->clear('extra_fields.display.language');
    }
    $extra_fields_form_langcode = $config->get('extra_fields.form.language');
    if (isset($extra_fields_form_langcode)) {
      $config->set('extra_fields.form.langcode', $extra_fields_form_langcode);
      $config->clear('extra_fields.form.language');
    }
```